src/code/z_kaleido_setup.c

```diff
@@ -20,7 +20,7 @@ void KaleidoSetup_Update(PlayState* play) {
         (play->sceneId != SCENE_BOMBCHU_BOWLING_ALLEY || !Flags_GetSwitch(play, 0x38))) {
 
         if (CHECK_BTN_ALL(input->cur.button, BTN_L) && CHECK_BTN_ALL(input->press.button, BTN_CUP)) {
-            if (BREG(0)) {
+            if (OOT_DEBUG && BREG(0)) {
                 pauseCtx->debugState = 3;
             }
```

Comment:
interesting that this is here. retail code is still checking for this button combo, but then just doing nothing?
I guess this kinda supports the "debug reg macro getter" theory